manifests/01-role-binding-authentication-reader.yaml

```diff
@@ -0,0 +1,14 @@
+apiVersion: rbac.authorization.k8s.io/v1
+kind: RoleBinding
+metadata:
+  creationTimestamp: null
+  name: foo-role
+  namespace: kube-system
+roleRef:
+  apiGroup: rbac.authorization.k8s.io
+  kind: Role
+  name: extension-apiserver-authentication-reader
+subjects:
+- kind: ServiceAccount
+  name: cluster-storage-operator
+  namespace: openshift-cluster-storage-operator
```

Comment:
Do we need rbac permissions to read/update `storage` object CR? Also - I must have missed something but I can't find a place where `cluster` object is created.

Reply:
good catch, I forgot to push the last commit. It's created by CVO now.